packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -276,6 +276,7 @@ tags=$(
   IFS=,
   echo "${agent_metadata[*]}"
 )
+endpoint=${BUILDKITE_AGENT_ENDPOINT}
```

Comment:
```suggestion
endpoint=${BUILDKITE_AGENT_ENDPOINT:-"https://agent.buildkite.com/v3"}
```

As an extra level of precaution, would it be a good idea to re-iterate the default value here?

Reply:
Great suggestion, commit added 👍🏻